controllers/provider-aws/charts/provider-aws/values.yaml

```diff
@@ -37,3 +38,8 @@ config:
       ami: ami-07739b17529e8c1d0
     - name: ap-southeast-2
       ami: ami-02d7d488d701a460e
+  etcd:
+    storageClass: gardener.cloud-fast
```

Comment:
To avoid confusion I would also change the values API to match the new ConfigMap API.

Reply:
Done.